### PR TITLE
Issue #336 Replacing Utils.move_file with Shutil.move function

### LINKED ISSUE #336: Replace utils.move_file() with shutil.move()
## Summary

I think that the function [`utils.move_file()`](https://github.com/physiopy/phys2bids/blob/da3a5306a91a046f3956f1d033b6f86b5c14611b/phys2bids/utils.py#L152) could be replaced with `shutil.move()`, which automatically overwrites existing files, as desired in `utils.move_file()`.

## Next Steps

    * Remove the function and associated tests
    * Remove the item from the API docs page
    * Replace the function call throughout the package

### PR DESCRIPTION
**Pull request by jiyoungsin**
Closes #336

## Proposed Changes

  - Removed the function and associated tests DONE.
  - Remove the item from the API docs page NONE.
  - Replace the function call throughout the package DONE.

### CODE REVIEW

**Comment by eurunuela** [comment]
Thank you so much @jiyoungsin !

I think we can just remove the function and the unit test. The function is not used anywhere in the code but the unit test anyway.

File: phys2bids/tests/test_utils.py [hunk 1]
```diff
@@ -66,7 +67,7 @@ def test_move_file(tmpdir):
         pass
```
Comment (smoia) [inline phys2bids/tests/test_utils.py:67]: I can't comment at the right line, but I would equally remove this chunk of code (lines 62-70)

**Review by smoia — CHANGES_REQUESTED** [review]
Hey @jiyoungsin , thank you for your first contribution!
I know it might look like a buzzkill now because in the end we should remove the function, but cleaning up code is a nice contribution nonetheless!
If you want to try again with a good first issue that is a tad more challenging, why don't you take #334 or #335?

We also need to slightly retouch the PR: I'm going to assign the correct label for your contribution, but in the meantime, could you slightly change the title to make it more "changelog ready"? You can have a look at [this release](https://github.com/physiopy/phys2bids/tree/2.2.0) for _a lot_ of examples, or in general at the release page.
Something like "Remove redundant utility function `move_file()`" could work.

Finally, the tests are failing (bad sign), but as soon as you address the changes I'm suggesting I'm sure it will work!

**Comment by smoia** [comment]
@jiyoungsin let us know if you need any help!

**Comment by jiyoungsin** [comment]
hey @smoia thanks again for waiting around for me, however, I have a lot of school work I must catch up on. Can you please assign this to someone else?

**Comment by eurunuela** [comment]
I assigned it to myself but been quite busy in the last few weeks. I'm thinking that this could be a good way to start working on phys2bids for some of the newcomers from Brainhack Donostia. What do you think @smoia ?

**Comment by smoia** [comment]
Indeed!

**Comment by smoia** [comment]
:rocket: Issue was released in [`2.3.0`](https://github.com/physiopy/phys2bids/releases/tag/2.3.0) :rocket:

**Pull request closed without merging**